app/models/camaleon_cms/post_comment.rb

```diff
@@ -1,7 +1,15 @@
 module CamaleonCms
   class PostComment < CamaleonRecord
     include CamaleonCms::Metas
-    include CamaleonCms::CommonRelationships
+
+    def self.inherited(subclass)
+      super
+
+      # instance_eval so the code is eval'd in the context of subclass, not subclass's class, which would be Class 🤪
```

Comment:
I think this change is a mistake. `PostComment` needs `CommonRelationships` and isn't inherited by other classes.

Reply:
Yes, thank you!

Fixed in https://github.com/owen2345/camaleon-cms/pull/1046/commits/78dc53c2d1a8a284516957452a7d4a5129709f62